.github/workflows/release.yml

```diff
@@ -36,6 +36,11 @@ jobs:
         run: npm run build
         working-directory: packages/wrangler
 
+      - name: SourceMapping Rewrite & Sentry Upload
```

Comment:
I think we really need a big comment, either here or in `sourcemap-url.js` explaining why this is necessary.